### PR TITLE
Update logo license and copyright file

### PR DESCRIPTION
**Pull request by RadenTheFolf**
Updated logo copyright info in COPYRIGHT.txt
Updated LOGO_LICENSE.txt to reflect current copyright info

supersedes #737

### CODE REVIEW

**Comment by SkogiB** [comment]
We can update the copyright with Joey's legal name, "Asrorul Irsyad"

File: COPYRIGHT.txt [hunk 1]
```diff
@@ -50,12 +50,19 @@ License: Expat
 
 Files: ./icon.png
  ./icon.svg
- ./logo.png
- ./logo.svg
-Comment: Redot Engine logo
+Comment: Redot Engine Icons
 Copyright: 2024, dev_airsyad (on X/Twitter)
```
Comment (Spartan322) [inline COPYRIGHT.txt]:
```suggestion
Copyright: 2024, Asrorul Irsyad
```

File: COPYRIGHT.txt [hunk 2]
```diff
@@ -50,12 +50,19 @@ License: Expat
 
 Files: ./icon.png
  ./icon.svg
```
Comment (Spartan322) [inline COPYRIGHT.txt]:
```suggestion
Files: ./icon.png
 ./icon.svg
 ./logo.png
 ./logo.svg
 ./logo_outlined.png
 ./logo_outlined.svg
 ./editor/icons/TitleBarLogo.svg
```

File: COPYRIGHT.txt [hunk 3]
```diff
@@ -50,12 +50,19 @@ License: Expat
...
 
+Files: ./logo.png
+ ./logo.svg
+ ./logo_outlined.png
+ ./logo_outlined.svg
+ ./editor/icons/TitleBarLogo.svg
+Comment: Redot Engine Logos
+Copyright: 2024, Asrorul Irsyad
+License: CC-BY-4.0
+
```
Comment (Spartan322) [inline COPYRIGHT.txt]:
```suggestion
```

**Review by Spartan322 — APPROVED** [review]

**Review by SkogiB — APPROVED** [review]

**Pull request merged**